src/applications/personalization/profile360/util/helpers.jsx

```diff
@@ -31,23 +31,6 @@ export const formTitles = Object.keys(formBenefits).reduce((titles, key) => {
   return titles;
 }, {});
 
-export const formDescriptions = Object.keys(formBenefits).reduce(
```

Comment:
There's some duplicated logic here and in `src/applications/personalization/dashboard/helpers.jsx`, any preference regarding how we consolidate these, could we consolidate these as a helper file within `/personalization`? @ncksllvn